client/src/app/+about/about-instance/contact-admin-modal.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-template #modal>
   <div class="modal-header">
-    <h1 i18n class="modal-title">Contact {{ instanceName }} administrator</h1>
+    <h1 i18n class="modal-title">Contact instance administrators</h1>
```

Comment:
Why? Is the title too long with the instance name?

Reply:
Sorry, I forgot to explain this part. It was indeed to get a shorter title, and to keep the information that you contact the administrator(s) of the instance; the action button cannot tell precisely who you contact, I thought the modal title was intended to give more details without being too long.

Do you think we should keep the instance name and find something else like put `contact the administrator(s)` in a sub-title?

Reply:
> Do you think we should keep the instance name for example in a sub-title and name the main title contact the administrator(s)?

:+1: